fineract-provider/src/main/java/org/apache/fineract/commands/domain/CommandWrapper.java

```diff
@@ -38,6 +38,8 @@
     private final String json;
     private final String transactionId;
     private final Long productId;
+    private final Boolean status;
```

Comment:
@nikpawar89  what is the use this status field? avoid modifying command handler infrastructure code